ang/civicase/activity/actions/services/move-copy-activity-action.html

```diff
@@ -3,7 +3,10 @@
     <div class="form-group">
       <label class="col-sm-4 control-label">{{ model.ts('Select Case') }}</label>
       <div class="col-sm-8">
-        <input class="form-control" crm-entityref="{entity: 'Case'}" ng-model="model.case_id" />
+        <input class="form-control" crm-entityref="{entity: 'Case', api: {
+            'case_type_id.case_type_category': model.caseTypeCategoriesWithAccessToBasicCaseInformation[0],
+            params: {'case_id.case_type_id.case_type_category': {'IN': model.caseTypeCategoryIDsWithAccessToBasicCaseInformation }}
+          }}" ng-model="model.case_id" />
```

Comment:
is it possible to define the API call parameters in the model? This block is a bit hard to follow. We could also test this value if we move it to the model.